src/main/java/org/folio/marccat/resources/TenantAPI.java

```diff
@@ -25,29 +33,26 @@
   private TenantRefService tenantRefService;
 
   @PostMapping
-  public ResponseEntity<String> create(
-    @RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
-    @RequestHeader(Global.OKAPI_URL) String okapiUrl,
-    @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
-    @RequestBody TenantAttributes attributes
-  ) throws SQLException, IOException {
+  public ResponseEntity<String> create(@RequestHeader(Global.OKAPI_TENANT_HEADER_NAME) String tenant,
+      @RequestHeader(Global.OKAPI_URL) String okapiUrl, @RequestHeader(Global.OKAPI_TO_URL) String okapiUrlTo,
+      @RequestBody TenantAttributes attributes) throws SQLException, IOException {
     addHeaders(tenant, okapiUrl, okapiUrlTo);
     tenantService.createTenant(tenant, okapiUrl);
-    if(!okapiUrl.isEmpty())
+    if (!okapiUrl.isEmpty())
```

Comment:
Should there be an error response if, somehow, the `X-Okapi-Url` header is missing?

Reply:
I don't think so, because in the previous test the okapi url was empty. @carmentrazza do you prefer me to control it?

Reply:
Load data only works under okapi. It was put in control, the test API was in error because we access through the okapi utrl to the configuration module where we save the parameters of the marccat database.

Reply:
There is no need an error response.

Reply:
I guess that is fine, but if the API is always called via Okapi (as it should be), then why the need for this check at all? Our test code should always do the right thing (call the API with X-Okapi-Url set). When could this the `okapiUrl.isEmpty()` return `false`?

Reply:
The check was made because when the test made with the
the create () method of the TenantTest class fails for this reason we pass the empty url.
That way it doesn't go through this piece of code. 
We have to fix the problem then this check will be removed.
This code sends a call via Okapi to the configuration module to get the connection to the marccat database and fails because Okapi is not available during compilation.
The connection parameters to the marccat database are saved on the configuration module for each tenant.
Marccat has a separate relational postgres database for each tenant. It is not a single schema for each tenant.
I don't know if you can give us directions to solve this problem in order to perform this test.